functions.php

```diff
@@ -51,9 +52,33 @@ function getTeamData(array $finals): string {
         if ($final === []) {
             return 'Got data?';
         } else {
-            $team .= '<div class="winners-box">' .
-                        '<img src="'.$final['Image'].'" alt="Grandfinal team photo" class="winners-image">' . '<div class="match-stats"><h6 class="team-name">' . $final['Premier'] . '</h6><p>' . $final['Season'] . '</p><p>' . $final['Runner-Up'] . '</p><p>' . $final['Score'] . '</p></div></div>';
+            if($final['Delete'] == '0') {
+                $team .= '<div class="winners-box">' .
+                        '<img src="'.$final['Image'].'" alt="Grandfinal team photo" class="winners-image">' . '<div class="match-stats"><h6 class="team-name">' . $final['Premier'] . '</h6><p>' . $final['Season'] . '</p><p>' . $final['Runner-Up'] . '</p><p>' . $final['Score'] . '</p></div>' . '<form action="delete.php" method="POST"><input type="submit" value="Delete"/><input class="hidden" name="dataID" value="' . $final['id'] . '"></form></div>';
+            }   
         }
     } 
     return $team;
+}
+
+/**
+ * Targets the ID in the database
+ *
+ * @return void
+ */
+    function targetID() {
```

Comment:
PSR-12 says functions look like this:
```
function doThing()
{
    // code
}
```

Reply:
Curly brackets moved onto a new line at the start of each function